tslearn/svm.py

```diff
@@ -61,14 +73,14 @@ class TimeSeriesSVC(BaseSVC):
     coef0 : float, optional (default=0.0)
         Independent term in kernel function.
         It is only significant in 'poly' and 'sigmoid'.
-    probability : boolean, optional (default=False)
-        Whether to enable probability estimates. This must be enabled prior
-        to calling `fit`, and will slow down that method.
     shrinking : boolean, optional (default=True)
         Whether to use the shrinking heuristic.
+    probability : boolean, optional (default=True)
```

Comment:
Putting on False by default does fail a estimator check. One possibility would be to use `hasattr` to check if it has a `probability` keyword during the estimator checks and put that on True if it exists.

Reply:
`scikit-learn` changes the values of some hyperparameters for tests:
https://github.com/scikit-learn/scikit-learn/blob/a83c8311dfdbf74dea584d45c6f254bc8171054d/sklearn/utils/estimator_checks.py#L322-L400

It may be out of scope for this PR, but it might be good to have something similar in the future (some tests are not well suited for the default values of some hyperparameters)

Reply:
It's actually really not a lot of work to use this approach (we are already doing it from the shapelets for example):

In `check_estimator`:

```
    if hasattr(estimator, 'max_iter'):
        estimator.set_params(max_iter=10)
    if hasattr(estimator, 'total_lengths'):
        estimator.set_params(total_lengths=1)
```

We can just add another if.